packages/PHPStanStaticTypeMapper/TypeMapper/UnionTypeMapper.php

```diff
@@ -128,6 +128,11 @@ public function mapToPhpParserNode(Type $type, string $typeKind): ?Node
             return null;
         }
 
+        // null is already part of mixed
+        if ($nullabledType instanceof MixedType) {
+            return new Name('mixed');
```

Comment:
this should re-assign so validation of Node result based on latest php feature can be re-checked on `$this->phpStanStaticTypeMapper->mapToPhpParserNode()` on next line:

```suggestion
            $nullableType = new MixedType();
```